### PR TITLE
Added Analog Clock project using HTML, CSS, and JavaScript

### PR DESCRIPTION
**Pull request by ShubhanshuJha**
# Zero To Mastery - Webblocks 2022 📦📦

 ## PR contains:
- INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜

## Checklist
 Put an `x` in all the boxes that apply: <br>
 

- [] 📝 My change requires a change to the documentation.
- [ ] 🎀 I have updated the documentation accordingly.

 ## What types of changes does your code introduce?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
| icon.png  | Title goes here   |
| Image goes here  | Image goes here   |

### CODE REVIEW

**Comment by ShubhanshuJha** [comment]
Added a new project, and did every step as per the suggestion.

File: src/blocks/ShubhanshuJha/style.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    opacity: 0.8;
+
+}
+
+#hour {
+    height: 26%;
+    width: 1.5%;
+    top: 24.4%;
+    left: 49.48%;
+    /* opacity: 0.6; */
```
Comment (sophiabrandt) [inline src/blocks/ShubhanshuJha/style.css:40]: Please remove all outdate code (everything that's commented out) in this file.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please provide the license to the audio file you used, see [here](https://github.com/zero-to-mastery/webblocks-2022#%EF%B8%8F-rules):
> If you use images or audio files, please make sure that you are allowed to use these (don't violate copyright!) - please add a link to the license into the pull request template

**Comment by ShubhanshuJha** [comment]
Did the change as per the suggestion.

**Comment by sophiabrandt** [comment]
@ShubhanshuJha please provide answer to my comment above, otherwise we cannot merge your request, as we don't want to violate copyright.

**Comment by ShubhanshuJha** [comment]
Yes, I added the license to use the image file.
And, removed the music, as I didn't have the license to use it anywhere.

**Pull request merged**